saleor/graphql/order/types.py

```diff
@@ -506,7 +499,7 @@ def resolve_available_shipping_methods(root: models.Order, _info):
 
     @staticmethod
     def resolve_invoices(root: models.Order, info):
-        return root.invoices.ready()
+        return root.invoices.all()
```

Comment:
Do we want to return all invoices for any order? I think we should use similar logic as in `resolve_user` function in this type, so you can only fetch invoices if you're an admin user with proper permission or you're a customer and this is your order. Should we return the invoices for anonymous users as well? They'll get them by email and I don't think we'll render them in the storefront. What do you think @patrys?